arangod/IResearch/IResearchLinkMeta.cpp

```diff
@@ -613,7 +613,9 @@ bool IResearchLinkMeta::init(
     constexpr std::string_view kFieldName{StaticStrings::VersionField};
 
     auto const field = slice.get(kFieldName);
-    mask->_version = field.isNumber<uint32_t>();
+    if (!field.isNone()) {
```

Comment:
It is not parsing, Just a noexcept type check. Are you sure it caused problems? Or you mean there is a possibility that `isNumber<T>` return `true`, but `getNumber<T>` throws ?

Reply:
Yes, I couldn't execute arangodbtests anymore. Locally and Jenkins complained as well.
In case the field had the type ("non-representable-type"), it run into following code during testing and caused trouble there:

Reply:
![image](https://user-images.githubusercontent.com/1244163/151030711-1d81712d-8057-4931-a7a5-c4869426257e.png)
`3rdParty/velocypack/include/velocypack/Slice.h`

Reply:
But even if throw - there is a `catch(...)`. So the method still just returns false. What is the failing test name?

Reply:
Just tried the devel. All tests seem to pass on devel. We need to investigate this before the merge. Happy to jump on a call if necessary.

Reply:
Test has been: IResearchQueryTraversalTest
But just verified that this change is not required anymore, issue had been somewhere else but is now fixed. 
Reverted in: 6800acf85129a9609541cb67106ab5f04b6ca666